lib/lita/adapters/slack/message_handler.rb

```diff
@@ -41,7 +41,9 @@ def handle
 
```

Comment:
commenting here for threading:

Should we create a new branch off of Erik's branch so it has a name that's clearer regarding why we're using that particular branch?  I'm honestly not sure how confusing the branch name would be for future engineers working on this

Reply:
Actually, scratch that—I think it's better to stick with the changes you made to the [changebot repo](https://github.com/change/changebot/pull/41/files#diff-4220a334d4c3d81c674519e059c0a3a38dcc668d8afa679aeb8ff0f579df25d3R23) instead, and not make the changes here.  Am I missing a strong reason for making the no-op changes in this PR?

Reply:
I find it confusing, I imagine future engineers will as well.